nrf_security/doc/backend_config.rst

```diff
@@ -82,44 +52,44 @@ Feature support
 |             |                   | 256-bit key |
 +-------------+-------------------+-------------+
 
+.. note::
+   * The :ref:`nrf_security_backends_oberon` uses some functionality from Original Mbed TLS for AES operations.
```

Comment:
So does cryptocell? For AES GCM?

This note makes it sound like it's just nrf_oberon doing this.

Reply:
nrf_oberon has a specialized way of handling AES operations by only changing 4 functions for ECB operationssetkey encrypt, setkey decrypt, encrypt and decrypt. The rest of the implementation comes from Mbed TLS (i.e. CTR, CFB, OFB, XTS, CBC, CMAC, CCM, GCM are all provided in code by Mbed TLS).  This makes nrf_oberon special

Reply:
Fixed